packages/playwright-test/src/reporters/base.ts

```diff
@@ -523,6 +523,23 @@ function fitToWidth(line: string, width: number, prefix?: string): string {
   return taken.reverse().join('');
 }
 
+export function uniqueProjectIds(projects: FullProject[]): Map<FullProject, string> {
```

Comment:
This repeats ConfigLoader._assignUniqueProjectIds, can you extract it into a common util?

Reply:
I'll follow up